README.md

```diff
@@ -85,14 +96,14 @@ Qserv install customization is handled with [Kustomize](https://github.com/kuber
 
 ```sh
 # Install a qserv instance with default settings inside default namespace
-kubectl apply -k ./base --namespace='default'
+kubectl apply -k https://github.com/lsst/qserv-operator/overlays/local --namespace='default'
```

Comment:
what kind of cluster is "local" for? a local kind instance? or local at IN2P3? 🙃

Reply:
OH, this is for a local git repository? (matches the `--local` arg in the deploy script?)

Reply:
This is the default install for development workstation, renamed to "dev"